services/apps/test/apps/test_fixtures.clj

```diff
@@ -26,16 +15,7 @@
   (let [config-path (getenv "APPS_CONFIG_PATH" default-config-path)]
     (require 'apps.util.config :reload)
     (apps.util.config/load-config-from-file config-path {:log-config? false})
-    (f)))
-
-(defn with-service-uris [f]
-  (binding [jex-uri                (getenv "JEX_URI" default-jex-uri)
-            data-info-uri          (getenv "DATA_INFO_URI" default-data-info-uri)
-            notification-agent-uri (getenv "NOTIFICATION_AGENT_URI" default-notification-agent-uri)
-            iplant-groups-uri      (getenv "IPLANT_GROUPS_URI" default-iplant-groups-uri)
-            metadata-uri           (getenv "METADATA_URI" default-metadata-uri)
-            permissions-uri        (getenv "PERMISSIONS_URI" default-permissions-uri)]
-    (f)))
+    (metadata-client/with-metadata-base (config/metadata-base) (f))))
```

Comment:
This change compensates for the fact that the integration tests don't go through the Ring middleware.